src/main/java/com/google/sps/scheduler/Scheduler.java

```diff
@@ -77,10 +76,10 @@
       while (eventStart - lastEnd >= DEFAULT_DURATION_IN_MILLISECONDS &&
           tasksIterator.hasNext()) {
         DateTime startTime = epochToDateTime(lastEnd, timeZone);
+        Task task = tasksIterator.next();
+        task.setDue(startTime.toStringRfc3339());
```

Comment:
I thought this only supports date? Or can you pass a datetime string but it will truncate it?

Reply:
It stores the date and time also. When it is updated through the API, the time is automatically not considered.